examples/creatives/base_simid_creative.js

```diff
@@ -236,4 +247,14 @@ class BaseSimidCreative {
     const logMessage = data.args.message;
     console.log("Received message from player: " + logMessage);
   }
+
+  /**Sends message requesting a to resize creative based off of given resizeParameters
```

Comment:
* Sends message requesting TO (get rid of a)

Reply:
Good catch! Fixed